systems/registry/network/pkg/config.go

```diff
@@ -32,6 +32,9 @@ type Config struct {
 	Service          *uconf.Service
 	PushGateway      string `default:"http://localhost:9091"`
 	OrgName          string
+	Country          string `default:"Democratic Republic of the Congo"`
+	Language         string `default:"en"`
+	Currency         string `default:"usd"`
```

Comment:
I don't think we should not set default values at model level for these properties. Let's set them via config.

Reply:
Okay